src/styles/main-theme.ts

```diff
@@ -0,0 +1,17 @@
+export interface Theme {
+  fontFamily: string;
+}
+
+export interface ThemeProps {
+  theme: Theme;
+}
+
+export const mainTheme = {
+  fontFamily:
+    "Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif",
+};
+
+export const getTheme = ({ theme: themeProps }: ThemeProps) =>
+  themeProps as Theme;
+export const themeFontFamily = ({ theme: themeProps }: ThemeProps) =>
+  themeProps.fontFamily;
```

Comment:
Not sure if we get much value from these functions. The second function does not start with a verb, so looks like it’s a value rather than a getter.

[Styled-components examples](https://styled-components.com/docs/advanced) use `${props => props.theme.xyz}`, which is a few characters shorter but one abstraction less.